content/cloud/variables.md

```diff
@@ -0,0 +1,191 @@
+
+[variables]
+password = { required = true }
+
+[[component]]
+id = "pw-checker"
+source = "app.wasm"
+[component.trigger]
+route = "/..."
+[component.build]
+command = "spin py2wasm app -o app.wasm"
+[component.config]
+password = "{{ password }}"
+```
+
+## Using Variables in a Spin Application
+
+Now that we have defined our variables and surfaced them to our component, we are ready to implement our Spin application. The application should get the user-provided password from the body of the HTTP request, compare it to the expected password set in our configuration variable, and authenticate accordingly. We will use the Spin `spin_config` module to retrieve the value of the `password` variable. 
+
+<!-- @selectiveCpy -->
+```py
+from spin_http import Response
+from spin_config import config_get
+
+def handle_request(request):
+    password = request.body.decode("utf-8")
+    expected = config_get("password")
+    access = "denied"
+    if expected == password:
+        access = "accepted"
+    response = f'{{"authentication": "{access}"}}'
+    return Response(200,
+                    {"content-type": "application/json"},
+                    bytes(response, "utf-8"))
+
+
+```
+
+Build and run the application locally to test it out. We will use the [environment variable provider](../spin/dynamic-configuration.md#environment-variable-provider) to set the variable values locally. The provider gets the variable values from the `spin` process's environment, searching for environment variables prefixed with `SPIN_CONFIG_`.
+
+<!-- @selectiveCpy -->
+```bash
+$ spin build
+$ SPIN_CONFIG_EXPECTED_PASSWORD="123" SPIN_CONFIG_PASSWORD="123" spin up
+Logging component stdio to ".spin/logs/"
+
+Serving http://127.0.0.1:3000
+Available Routes:
+  pw-checker: http://127.0.0.1:3000 (wildcard)
+```
+
+Send a request to the application with the correct password in the body to authenticate successfully.
+
+<!-- @selectiveCpy -->
+```bash
+$ curl -d "123" http://127.0.0.1:3000
+{"authentication": "accepted"}
+```
+
+## Deploying the Application to Fermyon Cloud
+
+Let's deploy our application to the cloud with initial values for our variables. All values are stored encrypted.
+
+<!-- @selectiveCpy -->
+```bash
+$ spin deploy --variable expected_password="123" --variable password="123"
+Uploading pw_checker version 0.1.0+r7123456...
+Deploying...
+Waiting for application to become ready........... ready
+Available Routes:
+  pw-checker: https://pw-checker-abcdefg.fermyon.app (wildcard)
+```
+
+Check if we can authenticate again using the deployed application:
+
+<!-- @selectiveCpy -->
+```bash
+$ curl -d "123" https://pw-checker-abcdefg.fermyon.app
+{"authentication": "accepted"}
+```
+
+We're in! Now, let's update one of the variables and observe how it dynamically changes. Spin provides a `cloud` plugin for managing Spin applications in Fermyon Cloud. The `variables` subcommand allows you to `set`, `list`, and `delete` application variables.
```

Comment:
"Spin provides a `cloud` plugin for managing Spin applications in Fermyon Cloud."

Maybe change to:
"Fermyon provides a plugin-in for the Spin CLI, for you to manage Spin applications in Fermyon Cloud."